app/assets/stylesheets/components/_c-card.scss

```diff
@@ -1,4 +1,15 @@
+%step {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: auto;
+  height: rem(3px);
+  background: $white;
+}
+
```

Comment:
@dhakelila yes! when you set the class "-card-steps" at the same level of "c-card" the steps will be appeared